clustered/build.gradle

```diff
@@ -16,6 +16,7 @@
 
 ext {
     coordinatorVersion = '1.0-SNAPSHOT'
+    offheapResourceVersion = '1.0-SNAPSHOT'
```

Comment:
This is expected to be the same version as above - that is terracotta-platform version. Can you properly indicate that?